docs/versioning_vendor_api.md

```diff
@@ -1,301 +1,451 @@
-# Versioning Vendor API
+# Vendor API Minor Versions
 
 ## Summary
 
-This document describes the steps required in order to increment the minor version of the Vendor API. Incrementing the API to major versions is currently not supported.
+This document describes the steps required in order to increment the minor version of the Vendor API.
 
+Incrementing the API to major versions is currently not supported.
 
-## Introducing new changes
+Minor version updates are only for non-breaking changes. A key design principle is maintaining backwards compatibility with all prior minor versions.
 
-Changes in the API are specified in `VersionChange` classes and each version can be made up of one or more classes. Every change class must be configured in `app/lib/vendor_api.rb`
+When incrementing a minor version, the following changes can be introduced:
+ - Adding fields to an existing schema
+ - Adding a new action to an existing controller
+ - Adding a new resource
+ - Adding metadata fields
```

Comment:
we don't have a section to explain metadata, do we want to add one?

Reply:
spoke in person:
1) Worth adding a line for the metadata section
2) Add an explanation for v1.1 spec and adding to it incrementally vs the draft

Reply:
Changes made - the docs bit has been expanded on, and there's a section that sets out the metadata stuff in some detail